rules/copyright.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    meta: {
+        fixable: "code",
+        schema: [
+            { type: "string" }
+        ]
+    },
+    create: function (context) {
+        const code = context.getSourceCode();
+        return {
+            Program(node) {
+                const firstToken = code.getFirstToken(node, { includeComments: false });
+
+                if (!firstToken) {
+                    return;
+                }
+
+                const headComments = code.getCommentsBefore(firstToken || node);
+                const hasSomeCopyrightHeader = headComments?.some(comment => comment?.value?.includes('Copyright'));
```

Comment:
This matching looks pretty dumb, but there are a few incarnations of the copyright header in the web repos and this seems to work.